### PR TITLE
run rustfmt on rtstartup folder

### PR DESCRIPTION
**Pull request by srinivasreddy**

### CODE REVIEW

File: src/rtstartup/rsbegin.rs [hunk 1]
```diff
@@ -40,7 +39,7 @@ pub mod eh_frames
 
     // Unwind info registration/deregistration routines.
     // See the docs of `unwind` module in libstd.
-    extern {
+    extern "C" {
```
Comment (GuillaumeGomez) [inline src/rtstartup/rsbegin.rs]: Why adding "C" in here?

Comment (Manishearth) [inline src/rtstartup/rsbegin.rs]: extern is extern "C" by default

**Comment by srinivasreddy** [comment]
cc @nrc

**Pull request merged**